docs/book/v1/intro.md

```diff
@@ -24,13 +24,25 @@ In order to implement the OAuth2 server, we first need to configure it. The
 first step is to generate new cryptographic keys. We need to execute the script
 `./vendor/bin/generate-oauth2-keys` in order to generate these keys.
 
+This script will store the keys in the application's `data` folder, if found:
+
 ```bash
 $ ./vendor/bin/generate-oauth2-keys
-```
 
-This script will store the keys in the application's `data` folder if found:
+This script is provided as a convenient way to generate keys for
+the OAuth2 server provider. You may choose instead to use an
+alternative method. For more information, see the install docs:
+https://oauth2.thephpleague.com/installation/
+
+Best available location for keys:
+./data
+You'll likely want to move them to a better location
+
+We'll put them in a subdirectory:
+./data/oauth
```

Comment:
This section is misleading: "Best available location", "better location", "We'll put them in a subdirectory" ?!
Instead of "best", we can suggest a "typical" location for the keys and reduce the section to one folder, instead of 3 options, like `./data/oauth`.

Reply:
That's the output of the command...

Reply:
This must be clear:

1. add the command which must be executed
2. label the output

Both must be separate from each other!

Reply:
The markdown convention is to use the dollar sign to distinguish them. If we don't see value on that, we must simply drop all dollar signs.

I'm good with either, honestly.

Reply:
> The markdown convention is to use the dollar sign to distinguish them.

The reader and user of this description does not need to know anything about Markdown. Therefore it is correct to remove the dollar signs.
It must be clear to the reader which command is to be executed and what the output of this command is – _without to know any other languages or syntax._